plutus-benchmark/flat/Codec.hs

```diff
@@ -15,50 +16,62 @@ import qualified Data.ByteString.Lazy               as LBS
 import           Data.Text                          (Text)
 
 import           Language.PlutusCore                (DefaultFun (..))
+import           Language.PlutusCore.CBOR           (InvisibleUnit (..))
 import           Language.PlutusCore.Universe
-import           Language.UntypedPlutusCore
+import           Language.UntypedPlutusCore         hiding (OmitUnitAnnotations, restoreUnitAnnotations)
 
 import           "pure-zlib" Codec.Compression.Zlib as PureZlib
 import           "zlib" Codec.Compression.Zlib      as Zlib
 import           Codec.Serialise                    (Serialise, deserialise, serialise)
 import           Flat                               (Flat, flat, unflat)
 
-type Tm a = Term a DefaultUni DefaultFun ()
+type Tm name = Term name DefaultUni DefaultFun ()
 
-data Codec a = Codec
-  { serialize   :: a -> BS.ByteString
-  , deserialize :: BS.ByteString -> a
+data Codec name = Codec
```

Comment:
I do not understand why you used the same `name` here and for the Term. In the term case it is used as the type of names in PLC ASTs, but in this case it's a type that is convertible to `ByteString`.

Reply:
Ah, I meant to change that back.  I kept getting confused because I was thinking of `a` as an annotation type so I temporarily changed it.